topics/single-cell/tutorials/bulk-deconvolution-evaluate/tutorial.md

```diff
@@ -0,0 +1,585 @@
+
+**Note: We are using the original imported data here, not the transposed data or collections.**
+
+> <hands-on-title>Build the Expression Set object</hands-on-title>
+>
+> 1. {% tool [Construct Expression Set Object](toolshed.g2.bx.psu.edu/repos/bgruening/music_construct_eset/music_construct_eset/0.1.1+galaxy3) %} with the following parameters:
+>    - {% icon param-file %} *"Assay Data"*: `EMTABesethealthy.expression.tabular` (Input dataset)
+>    - {% icon param-file %} *"Phenotype Data"*: `EMTABesethealthy.phenotype.tabular` (Input dataset)
+>
+>    > <comment-title></comment-title>
+>    >
+>    > An ExpressionSet object has many data slots, the principle of which are the experiment data (*exprs*), the phenotype data (*pData*), as well metadata pertaining to experiment information and additional annotations (*fData*).
+>    {: .comment}
+>
+{: .hands_on}
+
+Similar to the expression data, this ExpressionSet object needs to be duplicated 20 times into a collection for later batch processing.
+
+> <hands-on-title>Generate ESet collection</hands-on-title>
+>
+> 1. {% tool [Duplicate file to collection](__DUPLICATE_FILE_TO_COLLECTION__) %} with the following parameters:
+>    - {% icon param-file %} *"Input Dataset"*: `RData ESet Object` (output of **Construct Expression Set Object** {% icon tool %})
+>    - *"Size of output colection"*: `20`
+>
+> 2. **Rename** {% icon galaxy-pencil %} output `ESet Object`
+>
+{: .hands_on}
+
+
+# Create pseudo-bulk and actual cell proportions
+
+We are now going to run our first workflow! This workflow will extract a subsample from the data containing 200 random cells. The workflow will then perform two things with this subsample:
+
+1. Count the cell types and proportions of the data in order to be used as reference later against the predicted proportion values
+2. Remove the cell types and convert the single-cell data into pseudo-bulk data to be later inputted into the deconvolution tools.
+
+The above will be done twice to emulate multiple "subjects". Since the deconvolution tools will be expecting the bulk-RNA data to comprise of at least 2 subjects (each with their own bulk data). For this tutorial our subjects will simply be called **A** and **B**. However, in the real world these subjects could be different patients, tissue samples, diseased/healthy, etc. 
+
+> <comment-title>Different Results</comment-title>
+> Note that since we are selecting 20 samples, each containing 200 randomly selected cells. The plots and results presented in this tutorial will differ from your own. There will be some similarities such as certain cells being in higher proportion to others but the exact values with differ!
+{: .comment}
+
+**Remember** since we have a collection of 20 inputs, the output of this workflow will be a collection of 20 elements, each corresponding to the input elements. Each output will have its own random selection of 200 cells.
+
+> <comment-title>Inputting multiple datasets</comment-title>

```

Comment:
this could probably be an FAQ. or maybe you could use the existing ["select multiple datasets" faq](https://training.galaxyproject.org/training-material/faqs/galaxy/tools_select_multiple_datasets.html) faq? And maybe enhance it with your screenshot?